tests/integrationTests/tests/python_linehighlight/__init__.py

```diff
@@ -49,6 +50,15 @@ def initialize(test):
 
 ############################################################################
 
+@testcase
+def schema_validation(test):
+    config_path = os.path.join(test.testcase_path, 'assignment_config', 'complete_config.json')
+    schema = os.path.join(SUBMITTY_INSTALL_DIR, 'bin', 'json_schemas', 'complete_config_schema.json')
+    try:
+        submitty_schema_validator.validate_complete_config_schema_using_filenames(config_path, schema, show_warnings=False)
+    except submitty_schema_validator.SubmittySchemaException as s:
+        s.print_human_readable_error()
+        raise
```

Comment:
extract this block of code into a function in `lib.py` and just call that with just a parameter of the config_path instead of having this just be copied and pasted in each.

Reply:
Fixed in upcoming commit.